piet-cairo/src/blurred_rect.rs

```diff
@@ -0,0 +1,57 @@
+//! Helpers for efficiently drawing blurred rectangles.
+//!
+//! This function is not cairo-specific, but currently cairo is the only back-end
+//! that requires it, as other back-ends have their own implementation.
+//!
+//! As a future performance optimization, we might implement a cache of the computed
+//! images.
+
+use cairo::{Format, ImageSurface};
+
+use piet::kurbo::{Point, Rect};
+
+/// Extent to which to expand the blur.
+const BLUR_EXTENT: f64 = 2.5;
+
+pub fn compute_blurred_rect(rect: Rect, radius: f64) -> (ImageSurface, Point) {
```

Comment:
Does this need to be public? Feels like it is currently an implementation detail of other public API?